mlrun/model_monitoring/application.py

```diff
@@ -35,25 +36,28 @@ class ModelMonitoringApplicationResult:
     """
     Class representing the result of a custom model monitoring application.
 
-    :param application_name:     (str) Name of the model monitoring application.
-    :param endpoint_id:          (str) ID of the monitored model endpoint.
-    :param schedule_time:        (pd.Timestamp)Timestamp of the monitoring schedule.
-    :param result_name:          (str) Name of the application result.
-    :param result_value:         (float) Value of the application result.
-    :param result_kind:          (ResultKindApp) Kind of application result.
-    :param result_status:        (ResultStatusApp) Status of the application result.
-    :param result_extra_data:    (dict) Extra data associated with the application result.
+    :param application_name:      (str) Name of the model monitoring application.
+    :param endpoint_id:           (str) ID of the monitored model endpoint.
+    :param start_processing_time: (pd.Timestamp) Start time of the monitoring schedule.
```

Comment:
so start_processing_time would equal last_period_analyzed per the specific app?
end_processing_time = last_processing_time = last_ready_period + base_period?
need to be consistent if the ranges are  "open" or "close" (ie do they include the limit points themselves)